streamer/src/nonblocking/quic.rs

```diff
@@ -185,8 +224,37 @@ async fn run_server(
         stats.clone(),
         coalesce,
     ));
+
+    let mut accepts = incoming
```

Comment:
Please add some comments please on what we are trying to achieve.

Reply:
@alessandrod

Reply:
```suggestion
	// Create the set of accept() futures we're going to listen on for connections. We wrap quinn's
	// incoming.accept() so we can store the endpoint index in EndpointAccept::endpoint. When a
	// future resolves, we use the index to call incoming[index].accept() again. This way we don't
	// have to recreate the whole set of accept() futures at every loop iteration, saving wakeups
	// and tokio runtime work.
    let mut accepts = incoming
```